server/passport-providers.js

```diff
@@ -159,5 +159,20 @@ module.exports = {
     scope: ['email'],
     link: true,
     failureFlash: true
+  },
+  'github-repo': {
```

Comment:
Yup I agree. 

Plus this is the only we way we can ask interested users (currently already in the database and linked with Github) for escalation in the scope access from `email` to `email + repo`

Nevertheless this is going to use the same application id and secret as the current app.